### DISCUSSION TITLE
Welcome to the Enclave Workshops Discussions!

### DISCUSSION THREAD

**Discussion by jkfitzsimons** [ObliviousAI/enclave_workshops#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ben-santos** [comment]
Hello, I am Ben. I’ve been working on PETs since I joined StatCan as Data Scientist in 2021 :rocket:

We explored (and continue exploring) PETs (Homomorphic Encryption, Federated Learning, Private Set Intersection, Secure Multi-Party Computation) on different proof-of-concepts.

I am interested in enclaves since they are not orthogonal to the technologies we use and could provide an extra layer of confidentiality and security. I think this hands-on it's a very interesting way to get started on enclaves.

Thanks,

**Reply by jkfitzsimons** [reply]
Thank you, @ben-santos!

It would be great to hear if there were any specific topics or examples that you think would be useful. Happy to be very flexible on this end with regard to content 😄
